terraform/modules/env_baseline/main.tf

```diff
@@ -305,6 +305,11 @@ resource "google_cloud_run_service" "default" {
           value = var.github_app_installation_url
         }
 
+        env {
+          name  = "DISABLE_EVENT_TRACKING"
+          value = 0
```

Comment:
I think we should disable this event for our deployments - not to be mixed with real data

Reply:
sure this was for testing before I will merge I will change it to 1